app/src/main/java/com/example/fundatecheroes/profile/view/ProfileActivity.kt

```diff
@@ -0,0 +1,101 @@
+package com.example.fundatecheroes.profile.view
+
+import android.content.Intent
+import android.content.res.ColorStateList
+import android.os.Bundle
+import android.os.Handler
+import android.view.LayoutInflater
+import android.widget.Button
+import android.widget.EditText
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import com.example.fundatecheroes.R
+import com.example.fundatecheroes.databinding.ActivityLoginBinding
+import com.example.fundatecheroes.databinding.ActivityProfileBinding
+import com.example.fundatecheroes.home.view.HomeActivity
+import com.google.android.material.snackbar.Snackbar
+
+class ProfileActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityProfileBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityProfileBinding.inflate(layoutInflater)
+
+        getSupportActionBar()?.hide()
+        setContentView(binding.root)
+
+        val editTextN1 = binding.onboardingNome
+        val edittextN2 = binding.onboardingEmail
+        val edittextN3 = binding.onboardingSenha
+
+        binding.buttonCriarConta.setOnClickListener {
+            validacaoPreenchimento(editTextN1, edittextN2, edittextN3)
+        }
```

Comment:
Podemos fazer da seguinte forma
```suggestion
binding.run{
       buttonCriarConta.setOnClickListener {
            validacaoPreenchimento(onboardingNome, onboardingEmail, onboardingSenha)
        }
}
```